lib/kasket/relation_mixin.rb

```diff
@@ -2,9 +2,13 @@ module Kasket
   module RelationMixin
     def to_kasket_query(binds = nil)
       if arel.is_a?(Arel::SelectManager)
-        arel.to_kasket_query(klass, (binds || bind_values))
+        if ActiveRecord::VERSION::MAJOR < 5
+          arel.to_kasket_query(klass, (binds || bind_values))
+        else
+          arel.to_kasket_query(klass, (@values[:where].binds.map(&:value_for_database) + Array(@values[:limit])))
```

Comment:
maybe blame bind_values inide of the old version of arel and see what they did when they removed the method